package/cpp/api/JsiSkContourMeasure.h

```diff
@@ -19,22 +18,24 @@ namespace RNSkia {
     class JsiSkContourMeasure : public JsiSkWrappingSkPtrHostObject<SkContourMeasure> {
     public:
         JsiSkContourMeasure(std::shared_ptr<RNSkPlatformContext> context,
-                  const sk_sp<SkContourMeasure> contourMeasure)
-                : JsiSkWrappingSkPtrHostObject(context, contourMeasure){};
+                            const sk_sp<SkContourMeasure> contourMeasure)
+                : JsiSkWrappingSkPtrHostObject(context, contourMeasure) {
+            assert(contourMeasure != nullptr);
```

Comment:
Shouldn't we throw a JS error - this assert will just make the app unexpectedly quit?

Reply:
Thankfully the only call to this constructor checks for null pointer. but update the code to throw.